benches/bench.rs

```diff
@@ -112,10 +114,15 @@ fn cdef_frame(c: &mut Criterion) {
 }
 
 fn cdef_frame_bench(b: &mut Bencher, w: usize, h: usize) {
-  let config =
-    EncoderConfig { quantizer: 100, speed_settings: SpeedSettings::from_preset(10), ..Default::default() };
+  let config = EncoderConfig {
+    width: w,
```

Comment:
Maybe name the parameters width and height so you can just do width, height, here?